ngraph_bridge/ngraph_mark_for_clustering.cc

```diff
@@ -781,6 +1165,11 @@ Status MarkForClustering(Graph* graph, const std::set<string> skip_these_nodes,
     }
   }
 
+  // Release backend created to query is_supported
+  if (ng_backend_type != "NOP") {
```

Comment:
why is it here, is the backend used somewhere else

Reply:
no, instead of doing it for each node doing it for all the nodes and then releasing outside the for loop.

Reply:
you can create outside the for loop too, it is confusing since you are creating inside, but releasing outside

Reply:
ya, could do that too.